api/v1beta2/multiclusterobservability_types.go

```diff
@@ -48,44 +40,161 @@ type MultiClusterObservabilitySpec struct {
 	ObservabilityAddonSpec *observabilityshared.ObservabilityAddonSpec `json:"observabilityAddonSpec,omitempty"`
 }
 
+type AdvancedConfig struct {
```

Comment:
do we only need provide the resource configuration for observabotirum deployment/statefulset? How about others such as grafana

Reply:
I think so far we don't have cases need to scale `grafana` and `alertmanager`. Do you have case for them?

Reply:
grafana always uses much more memory than most of other containers in our testing. It should have the scale requirement considering more clients.
No case for alertmanager yet since we have almost no workload on it in our test cases. But imo we'd better to have that if all of others expose resources as tweakable params.

Reply:
Agreed. Let me have subsequent PR to support grafana and alertmanager.

Reply:
ok